### ISSUE TITLE
Paywall remover not working

### ISSUE THREAD

**Issue by Shbhom** [AmoabaKelvin/Medium-Anti-Member-Only#8]
I'm using firefox 119.0.1 on arch based arco-linux
but still getting paywall banner even after using your extension
![image](https://github.com/AmoabaKelvin/Medium-Anti-Member-Only/assets/59741697/8b6d725f-5a31-45e7-a2ae-d44ff47f2af1)

**Comment by AmoabaKelvin** [comment]
Hello @Shbhom 
I will work on getting the firefox version updated, currently, the chrome version functions well

**Comment by malcolmmorton77** [comment]
Yeah, I just tried this and the problem is it's not appending the 12ft.io/ to the beginning of the article link. When I manually do it, I can read the article without the paywall banner but it doesn't actually provide anymore of the article. Do we need to reload the Firefox browser after installing the extension or immediately open a Medium article in the same session?
